app/design/frontend/Magento/luma/Magento_Newsletter/web/css/source/_module.less

```diff
@@ -102,3 +112,9 @@
         }
     }
 }
+
+
```

Comment:
Please, remove the extra lines here as well

Reply:
Hi,
I have remove the extra lines from app/design/frontend/Magento/luma/Magento_Newsletter/web/css/source/_module.less.
Please check.

Thanks,
Dipti